_site/css/bootstrap.css

```diff
@@ -7584,3 +7584,86 @@ hr.style-two {
   margin-top: 40px;
   margin-bottom: 50px;
 }
+
+/***** XDrone Spy *****/
```

Comment:
Please correct the comment for it to display the correct name for the section. XDrone 2 in this case.